ddl/ddl_api.go

```diff
@@ -378,14 +401,8 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 				constraints = append(constraints, constraint)
 				col.Flag |= mysql.UniqueKeyFlag
 			case ast.ColumnOptionDefaultValue:
-				value, err := getDefaultValue(ctx, v, colDef.Tp.Tp, colDef.Tp.Decimal)
+				hasDefaultValue, err = setDefaultValue(ctx, col, v)
```

Comment:
Could we use `setDefaultValue` here? This function calls `checkDefaultValue`, but the original code doesn't call.

Reply:
I think adding `checkDefaultValue` here doesn't matter.

Reply:
Oh, my God, I will fix it.

Reply:
done. Already add test for this. PTAL